### PR TITLE
Update styles on calls list

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
Fixes https://github.com/rapidpro/rapidpro/issues/1393

**Comment by norkans7** [comment]
Calls 2021-01-27 12-37-41

File: templates/channels/channelevent_calls.haml [hunk 1]
```diff
@@ -17,13 +17,20 @@
           Found {{ results_count }} calls in last 90 days for <i>{{search}}</i>.
 
 -block message-list
-  %table.list-table.list_rsms_message.table.table-condensed{style: '{% if not org_perms.msgs.broadcast_send  %}margin-top:10px{% endif %}'}
+  %table.list.object-list.lined.selectable
```
Comment (rowanseymour) [inline templates/channels/channelevent_calls.haml:20]: are they selectable?

Comment (norkans7) [reply templates/channels/channelevent_calls.haml:20]: It is the only easy way to have the pointer cursor on hover

Comment (rowanseymour) [reply templates/channels/channelevent_calls.haml:20]: But is that what we want when the rows aren't selectable?

Comment (norkans7) [reply templates/channels/channelevent_calls.haml:20]: I think so as the rows I clickable to navigate to the contact page

Comment (rowanseymour) [reply templates/channels/channelevent_calls.haml:20]: Ah right yeah 👍

**Review by rowanseymour — APPROVED** [review]
Nice! About time we fixed this

**Pull request merged**